### PR TITLE
Feat: Improve DisburseMaturityModal information

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Improve the information we show to the user when disbursing maturity.

# Changes

* Improve message in the first step of DisburseMaturityModal.
* Add data in the second step of DisburseMaturityModal.

# Tests

* Add tests that the new information is present as expected.
* Add methods in relevant POs.

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary yet, it's covered by the disburse maturity changelog entry.

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UI:

Screenshot 2023-09-13 at 15 24 26
Screenshot 2023-09-13 at 15 24 33

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte [hunk 1]
```diff
@@ -59,13 +91,28 @@
       bind:percentage={percentageToDisburse}
       disabled={percentageToDisburse === 0}
     >
+      <div class="percentage-container" slot="description">
+        <span class="description">
+          <Html
```
Comment (dskloetd) [inline frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte:96]: I don't see any HTML tags in `disburse_maturity_description_1`. Why do you need to use the `<Html>` component?

File: frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte [hunk 2]
```diff
@@ -74,16 +121,75 @@
       on:nnsCancel={modal.back}
     >
       <Html
```
Comment (dskloetd) [inline frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte:123]: `disburse_maturity_confirmation_description` doesn't seem to have HTML. Do we need the `Html` component here?

File: frontend/src/lib/i18n/en.json [hunk 3]
```diff
@@ -600,6 +600,13 @@
     "view_active_disbursements_modal_title": "Active Disbursements",
     "view_active_disbursements_to": "To: ",
     "view_active_disbursements_maturity": "~$maturity Maturity",
+    "disburse_maturity_description_1": "Burn maturity to mint new $symbol tokens. In 7 days, new $symbol tokens will be minted and sent to the destination address.",
+    "disburse_maturity_description_2": "The quantity of new $symbol minted will depend on the maturity modulation function (see the <a href=\"https://wiki.internetcomputer.org/wiki/Maturity_modulation\" target=\"_blank\">wiki page</a>). Consult with tax and legal advisors with further questions, and proceed at your own risk.",
+    "disburse_maturity_amount": "Choose how much of the maturity available to disburse:",
+    "disburse_maturity_confirmation_percentage": "Maturity Disbursed (%)",
+    "disburse_maturity_confirmation_amount": "Maturity Disbursed (Number)",
```
Comment (dskloetd) [inline frontend/src/lib/i18n/en.json:607]: "(Number)" doesn't tell me anything. Can we remove this or find a better word?

File: frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte [hunk 4]
```diff
@@ -74,16 +121,75 @@
...
-      />
+            })}</span
+          >
+        </KeyValuePair>
+        <KeyValuePair>
+          <span slot="key" class="description"
+            >{$i18n.neuron_detail.disburse_maturity_confirmation_amount}</span
+          >
+          <span data-tid="confirm-amount" class="value" slot="value"
+            >{formatMaturity(maturityToDisburse)}
```
Comment (dskloetd) [inline frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte:144]: Don't we want the total maturity available also to be visible on this step so it's clear what this amount is a percentage of?

File: frontend/src/lib/i18n/en.json (hunk 3 above)
Comment (lmuntaner) [reply frontend/src/lib/i18n/en.json:607]: How about (Amount)?

This goes next to "Maturity Disbursed (%)"

File: frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte:96]: Good point! I copy pasted from the other open PR.

File: frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte (hunk 2 above)
Comment (lmuntaner) [reply frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte:123]: ditto. I copied. I'll change it.

File: frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte (hunk 4 above)
Comment (lmuntaner) [reply frontend/src/lib/modals/neurons/DisburseMaturityModal.svelte:144]: I'm not convinced. It's a lot of information we already show to the user in this screen.

I'll speak with Artem about it. But if we change something, I'd rather change it in another PR.

This design was already discussed between Max, Mary and Artem.

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/lib/i18n/en.json (hunk 3 above)
Comment (dskloetd) [reply frontend/src/lib/i18n/en.json:607]: "Amount" is better than "Number". But feel free to discuss with PM/UX first.

**Review by dskloetd — APPROVED** [review]

File: frontend/src/lib/i18n/en.json (hunk 3 above)
Comment (lmuntaner) [reply frontend/src/lib/i18n/en.json:607]: will do

**Pull request merged**